src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
To hell with C standards, the long-term goal should be to eventually put that programming language in the grave.

On a more serious note, replacing whitespace in imported code makes it more cumbersome to diff any real changes, and I think that's a pretty compelling argument for rejecting https://github.com/mmaker/ocaml-nocrypto/commit/159d297a7918740c97279f01e67350947f56918b

As far as https://github.com/mmaker/ocaml-nocrypto/commit/f02b69524967108bb326b68f2d055bb8324476f5 and https://github.com/mmaker/ocaml-nocrypto/commit/84d6a86e969eca6bf5d642f9e2a4ccc222e7bf1f go, one way to satisfy all parties would be to add a section to the Makefile that detects the platform and uses `sed` to replace the pragmas with if-guards in the unfortunate cases when someone is trying to use that. Either that, or recommend cross-compiling.